supabase/seed.sql

```diff
@@ -0,0 +1,26 @@
+DO $$
+DECLARE
+  i INT;
+  randStatus TEXT;
+  randLang CHAR(2);
+  randDescription TEXT;
+  randLocation GEOMETRY;
+  lon FLOAT;
+  lat FLOAT;
+BEGIN
+  FOR i IN 1..100 LOOP -- Generate 100 random rows (change range as needed)
```

Comment:
I'm going to change the default to 90. The first page of Supabase shows 100 rows by default so it took me a minute to realise added moments were actually going to the next page 😅